.github/workflows/verification.yml

```diff
@@ -51,19 +51,20 @@ jobs:
       - name: Proto-Datacatalog-Go
         run: python3 /usr/local/bin/entrypoint.py -i ./protos -d protos/flyteidl/datacatalog -l go --go_source_relative --validate_out
 
+      # Python is going to be generated using buf
       # Python
-      - name: Proto-Service-Python
-        run: python3 /usr/local/bin/entrypoint.py -i ./protos -d protos/flyteidl/service -l python
-      - name: Proto-Admin-Python
-        run: python3 /usr/local/bin/entrypoint.py -i ./protos -d protos/flyteidl/admin -l python
-      - name: Proto-Core-Python
-        run: python3 /usr/local/bin/entrypoint.py -i ./protos -d protos/flyteidl/core -l python
-      - name: Proto-Event-Python
-        run: python3 /usr/local/bin/entrypoint.py -i ./protos -d protos/flyteidl/event -l python
-      - name: Proto-Plugins-Python
-        run: python3 /usr/local/bin/entrypoint.py -i ./protos -d protos/flyteidl/plugins -l python
-      - name: Proto-Datacatalog-Python
-        run: python3 /usr/local/bin/entrypoint.py -i ./protos -d protos/flyteidl/datacatalog -l python
+      # - name: Proto-Service-Python
```

Comment:
should we remove it?

Reply:
This will all be removed once we move the other plugins to buf. Leaving it like this is a good indication that we're in this in-between state.